.github/workflows/push.yml

```diff
@@ -0,0 +1,146 @@
+name: GitHub Actions
+on: [ push ]
+concurrency: ci-${{ github.ref }} # to avoid tag collisions in the ECR
+env:
+  # Name of the image in the ECR
+  APP_IMAGE: oracle-transformation
+
+  # Developer portal login
+  KBC_DEVELOPERPORTAL_VENDOR: "keboola"
+  KBC_DEVELOPERPORTAL_APP: "keboola.oracle-transformation"
+  KBC_DEVELOPERPORTAL_USERNAME: "keboola+oracle_transformation_github_actions"
+  KBC_DEVELOPERPORTAL_PASSWORD: ${{ secrets.KBC_DEVELOPERPORTAL_PASSWORD }}
+
+  # DockerHub login
+  DOCKERHUB_USER: "keboolabot"
+  DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  # Oracle database credentials
+  ORACLE_DB_HOST: "oj-oracle-test.cvkpp0ld53zd.us-east-1.rds.amazonaws.com"
```

Comment:
to je schválně oj-oracle-test ?, jestli jo, tak to není úplně dobrej název resource

Reply:
jj pravda... vytvořil jsem novou db a tady to změnil - https://github.com/keboola/oracle-transformation/pull/8/commits/93e69ebf1328e6cf5ccbe0bc1530ad152b7ddf98